clients/fides-js/src/lib/consent-value.ts

```diff
@@ -23,16 +23,16 @@ export const resolveLegacyConsentValue = (
 
 export const resolveConsentValue = (
   notice: PrivacyNotice,
-  context: ConsentContext
+  context: ConsentContext,
+  previouslyConsented?: boolean
 ): boolean => {
-  if (notice.current_preference) {
-    return transformUserPreferenceToBoolean(notice.current_preference);
-  }
```

Comment:
this function now needs the cookie to determine this part, instead of looking at `notice.current_preference` right? I think `previouslyConsented` muddies up the intended logic here a bit

Reply:
yes I think you're right. good catch!